vllm_ascend/spec_decode/mtp_proposer.py

```diff
@@ -105,6 +106,11 @@ def load_model(self, model) -> None:
         process_weights_after_loading(self.model, draft_model_config,
                                       target_device)
 
+        # use main model's embedding and LMhead
```

Comment:
let' wait https://github.com/vllm-project/vllm-ascend/pull/3528 merge first. Then mtp change can be in torchair module as well.